src/main/scala/uk/gov/hmrc/hmrcfrontend/views/viewmodels/Service.scala

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.hmrcfrontend.views.viewmodels
+
+import play.twirl.api.Html
+import uk.gov.hmrc.govukfrontend.views.viewmodels.phasebanner.PhaseBanner
+
+case class Service(
```

Comment:
just created a case class for this to show what sort of things would probably remain fixed across a service - I guess this would be a singleton hydrated from config somewhere.